README.md

```diff
@@ -161,6 +179,33 @@ spec:
 
 This extra information is needed, because it cannot be inferred from the resource's information.
 
+_Note_: `resourceRef.Version` here must match the resource's meta version in order to create the right
+layer information and tag in our internal OCI repository. If there is a mismatch here, the helm deployer
+resource will not find the chart.
+
+A corresponding Helm FluxDeployer might look something like this:
+
+```yaml
+apiVersion: delivery.ocm.software/v1alpha1
+kind: FluxDeployer
+metadata:
+  name: fluxdeployer-podinfo-pipeline-backend
+  namespace: ocm-system
+spec:
+  interval: 1m0s
+  sourceRef:
+    kind: Resource
+    name: ocm-with-helm-deployment
+  helmReleaseTemplate:
+    chart:
+      spec:
+        chart: podinfo
+        version: 6.3.5 # this is the version that must match with the resource version.
+    interval: 5m
+```
+
+It's possible to leave out the version in which case `latest` is used. But that might cause problems down the line later on.
```

Comment:
Something feels off about suggesting that a version should be specified here.

If I look at my POC set up, I have
- Repo that defines component does _not_ contain ComponentVersion, Configuration, Localization or FluxDeployer and those resources are not part of the component.   ( Meaning none of these resources can e configured or localized )
-  ComponentVersion, Configuration, Localization and FluxDeployer are owned by whomever is managing/operating the k8s clusters.
- Whomever is managing/operating the cluster is setting up a pipeline of sources ComponentVersion -> Localization -> Configuration -> Localization.   
- Any version filtering is happening with in the ComponentVersion spec.   Everything else just accepts the latest thing that makes it through the ComponentVersion filter.

Anyhoo, with such a pipeline you really need FluxDeployer to not specify a chart version.    I could perhaps see a an exceptional situation where the cluster managers/operators decide that after version X they want to change how a chart is deployed, but that is really exceptional

Reply:
But you know what chart version you want to deploy before. It won't be just latest.